community/php7/APKBUILD

```diff
@@ -326,7 +324,7 @@ _build() {
 		--enable-zip=shared \
 			--with-libzip=/usr \
 		--with-zlib=shared \
-		--with-zlib-dir=/usr \
+			--with-zlib-dir=/usr \
```

Comment:
Do we need this line after https://github.com/alpinelinux/aports/pull/2951/files